meteor/client/ui/RundownView/RundownTiming/RundownTimingProvider.tsx

```diff
@@ -145,8 +151,12 @@ export const RundownTimingProvider = withTracker<
 			}
 		}
 
+		calmDownTiming = (time: number) => {
+			return Math.round(time / 40) * 40
```

Comment:
Maybe the `40` could be moved to a constant somewhere around line 20, how about

```
const CURRENT_TIME_GRANULARITY = 40
```
? (or 66, 67 or 70 or whatever we decide)

Reply:
Agree, I will add this 👍